src/main/interface/cli.c

```diff
@@ -1041,7 +1041,9 @@ static void cliSerialPassthrough(char *cmdline)
     }
 
     // Register control line state callback
-    serialSetCtrlLineStateCb(cliPort, cbCtrlLine);
+    if (serialPassthroughDtrPin != IO_NONE) {
```

Comment:
(personally, I prefer `if (serialPassthroughDtrPin)` - `IO_t` is guarantied to work in bool context)